### DISCUSSION TITLE
AMD Software: Adrenalin Edition 22.7.1 not work

### DISCUSSION THREAD

**Discussion by prophet233** [xanderfrangos/twinkle-tray#417] [Monitor Compatibility Issues]
Since the driver of the graphics card was updated, the software indicated that the display of "DDC / Ci" could not be found. 22.5.1 can be used.

**Comment by vrjohannes** [comment]
I really do not know why, but very often AMD drivers just come without full DDC/CI-support, this is strange, because DDC/CI is no new science and does not harm a driver in any way. I think it is just laziness to integrate the full functionality. Nvidia game ready drivers never had this issue, they always come with full DDC/CI-support.
